client_options.go

```diff
@@ -102,17 +102,13 @@ func (w *signRequest) Configure(c *WalletClient) {
 }
 
 // initializeAccessKey handles the specific initialization of the access key.
-func (w *accessKeyConf) initializeAccessKey() (*bec.PrivateKey, error) {
+func (w *accessKeyConf) initializeAccessKey() (*ec.PrivateKey, error) {
 	var err error
-	var privateKey *bec.PrivateKey
-	var decodedWIF *wif.WIF
+	var privateKey *ec.PrivateKey
 
-	if decodedWIF, err = wif.DecodeWIF(w.AccessKeyString); err != nil {
-		if privateKey, err = bitcoin.PrivateKeyFromString(w.AccessKeyString); err != nil {
-			return nil, errors.Wrap(err, "failed to decode access key")
-		}
-	} else {
-		privateKey = decodedWIF.PrivKey
+	privateKey, err = ec.PrivateKeyFromWif(w.AccessKeyString)
```

Comment:
Why don't to skip explicit declaration of `var err error & var privateKey *ec.PrivateKey`
And use `:=` instead:
```suggestion
	privateKey, err := ec.PrivateKeyFromWif(w.AccessKeyString)
```

Reply:
applied suggestion